black.py

```diff
@@ -56,6 +56,11 @@
 
 from _black_version import version as __version__
 
+# This is needed for MyPy static type analysis.
+# See https://stackoverflow.com/a/38962160/1354930
+if False:
```

Comment:
nit: using `TYPE_CHECKING` is a bit more expressive, so you don't need the above comment.
```suggestion
if typing.TYPE_CHECKING:
```

Reply:
Ah, good to know! I'll keep that in mind for the future, thanks!